tests/torch/composite/test_sparsity_quantization.py

```diff
@@ -52,3 +52,11 @@ def test_can_quantize_inputs_for_sparsity_plus_quantization():
 
     assert len(input_quantizer) == 1
     assert isinstance(list(input_quantizer.values())[0], SymmetricQuantizer)
+
+
+def test_compression_rate_for_sparsity_plus_quantization():
+    model = BasicConvTestModel()
+    config = get_basic_sparsity_plus_quantization_config()
+    register_bn_adaptation_init_args(config)
+    sparse_quantized_model, compression_ctrl = create_compressed_model_and_algo_for_test(model, config)
+    assert compression_ctrl.compression_rate == 0.0
```

Comment:
Did that test fail without fix? What was the actual value?

Reply:
Yes. Calling `.compression_rate` raised an exception: quantization controller returned None as compression rate and sparsity controller returned 0.0 and these two values can't be summed together.